loader-tests/utils.ts

```diff
@@ -1,63 +1,91 @@
+import {
+  enableTestTime,
+  getTestEnvironment,
+  setBaseTestRoute,
+  setRequestMethod,
+  setupEnvironment,
+  userId
+} from '@slowreader/core'
 import type { ReadableAtom } from 'nanostores'
-import { isAbsolute, join, resolve } from 'node:path'
+import { readFile } from 'node:fs/promises'
+import { isAbsolute, join } from 'node:path'
 import pico from 'picocolors'
 
-const supportedExtensions = ['.xml', '.opml']
-
-export function isSupportedExt(path: string): boolean {
-  return supportedExtensions.some(ext => path.endsWith(ext))
+export async function readText(path: string): Promise<string> {
+  let absolute = path
+  if (!isAbsolute(absolute)) {
+    absolute = join(process.env.INIT_CWD!, path)
```

Comment:
The tricky way to make it work from root and from `loader-test/` dir.

`INIT_CWD` env contains `process.cwd()` before `pnpm` will change it to run `check-opml`.